ocean/sediment/sed_tracer/100m/config_forward2.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<config case="forward2">
+	<add_link source="../init/ocean.nc" dest="init.nc"/>
+	<add_link source="../init/mesh.nc" dest="mesh.nc"/>
+	<add_link source="../init/graph.info" dest="graph.info"/>
+	<add_link source="../gotmturb.nml" dest="gotmturb.nml"/>
```

Comment:
```suggestion
	<copy_file source_path="script_test_dir" source="gotmturb.nml" dest="gotmturb.nml"/>
```
This link doesn't point to anything currently, and the test seems to fail as a result.  My suggestion is to make a local copy of the namelist.  This is what was done in `gotm` tests.  That way, if a user modifies the file, it doesn't change the version in the repo and accidentally get committed.

Reply:
I used `add_link` as you suggested in the following comment.